src/services/EdFi.DataManagementService.Api/Infrastructure/WebApplicationBuilderExtensions.cs

```diff
@@ -29,6 +32,27 @@ public static void AddServices(this WebApplicationBuilder webAppBuilder)
         webAppBuilder.Services.AddTransient<IDataModelProvider, DataModelProvider>();
         webAppBuilder.Services.AddTransient<IAssemblyProvider, AssemblyProvider>();
 
+        var issuer = webAppBuilder.Configuration.GetValue<string>("AppSettings:Authentication:Issuer");
+        var authority = webAppBuilder.Configuration.GetValue<string>("AppSettings:Authentication:Authority");
+        var signingKeyValue = webAppBuilder.Configuration.GetValue<string>(
+            "AppSettings:Authentication:SigningKey"
+        );
+        webAppBuilder
+            .Services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
+            .AddJwtBearer(options =>
+            {
+                options.Authority = authority;
+                options.TokenValidationParameters = new TokenValidationParameters
+                {
+                    ValidateLifetime = true,
+                    ValidateAudience = false,
+                    ValidIssuer = issuer,
+                    IssuerSigningKey = new SymmetricSecurityKey(Encoding.UTF8.GetBytes(signingKeyValue!))
+                };
+                options.RequireHttpsMetadata = false;
+            });
+        webAppBuilder.Services.AddAuthorization();
```

Comment:
This is taking the OAuth work much farther than the scope of the ticket. The scope was just to accept a POST request and return a string as a token. 

This might be fine. I need to think about it a little bit.

Reply:
After further thought, please remove this code.

There is a danger that people interacting with the DMS - including ourselves! - might get confused and think that we have implemented more security mechanisms than we really have yet.

For example, once we add authorization, we'll need to be much more specific about authorization rules than this.

Please also keep in mind the long-term vision is that third party software will be generating the JWT. The DMS will need to validate it, but not generate it.